### ISSUE TITLE
Allow Custom Tooltip Content for LineChart

### ISSUE THREAD

**Issue by jdoiro3** [tremorlabs/tremor-npm#242]
## Feature Request

Could you expose more Recharts [props](https://recharts.org/en-US/api/Tooltip) in your components. Specifically, I'd like to customize the tooltip content?

https://github.com/tremorlabs/tremor/blob/b7dc5b60d33a45eec5f4e640c449b78998bd9f58/src/components/chart-elements/LineChart/LineChart.tsx#L102-L110

**Comment by MubashirullahD** [comment]
I too desire a custom tool tip. I have a scenario where the x-axis content isn't the data I want to show.

Example scenario is I have two date range pickers and so the plot is a comparison between the two range. The data is fed as categories like

{
date: "01",
"Jan 1, 2022 - Feb 1, 2022": "$100",
"Jan 1, 2021 - Feb 1, 2021": "$80"
}
If the ranges selected map on each other with a year difference, I can feed date things like "Jan 1" etc and that looks cool on the tool tip but if it is a custom range (meaning the dates picked in one range do not map on the same month period on the other range picker), then the x-axis is absolute numbers like 1, 2,3. If I had a tool tip I could customize, I could show that 01 means Jan 1 for range 1 and Sep 1 for range 2 on the tool tip.

In other words, custom tool tip very important.

**Comment by mitrotasios** [comment]
hey all, for now it is not possible to configure the tooltip, but we aim to allow for more customisation in the near future (hopefully Q1/23)

**Comment by MubashirullahD** [comment]
Looking forward to [it](https://www.decisionfoundry.com/tableau-consulting/articles/tableau-hacks-plot-different-date-ranges-on-the-same-axis/). 
![image](https://user-images.githubusercontent.com/45071858/211263378-a10ad292-5b74-4cf3-9a54-ac55643ed7dd.png)

**Comment by williamlmao** [comment]
+1

**Comment by davialcantaraa** [comment]
Up

**Comment by feliciovcm** [comment]
Up

**Comment by htroisi** [comment]
+1

**Comment by christopherkindl** [comment]
@jdoiro3 / @htroisi / @davialcantaraa / @feliciovcm / @williamlmao Better late than never: https://lab.tremor.so/tooltip

Will be released in the next version. Let me know if you any feedback! 🙂

**Comment by jdoiro3** [comment]
https://camo.githubusercontent.com/18cdb04e93fe276412b588dbed23bc328614bf7a4d09ea4d8862d38f1c466c65/687474703a2f2f6d656469612e74756d626c722e636f6d2f74756d626c725f6c74757a6a766251364c31717a677078392e676966
